src/lib.rs

```diff
@@ -93,6 +95,7 @@ impl<R: Runtime> WindowExt for Window<R> {
       let PhysicalPosition { x, y } = self.outer_position()?;
       let maximized = self.is_maximized().unwrap_or(false);
       let visible = self.is_visible().unwrap_or(true);
+      let decorated = self.is_decorated().unwrap_or(true);
```

Comment:
```suggestion
      let decorated = self.is_decorated().unwrap_or_else(|| true);
```

Reply:
has been changed. but why?

Reply:
explained here https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or

Reply:
nvm, looks like clippy is happy with `unwrap_or`